### ISSUE TITLE
Move hazardEntryListCount in the correct template 

### ISSUE THREAD

**Issue by diego2013** [diego2013/CSR_Meteor#122]
The method hazardEntryListCount (and similar siblings) should in in the actual template (in this case `advancedDetailsHazards`) instead of being in the scenarioCompleteForm helpers, where they are now.

By doing this whoever uses the template ca use this method (which might be useful given the nature of the template), as opposed to rewrite/duplicate the method.

**Comment by diego2013** [comment]
Instead of moving them to the `advancedDetailsHazards` (or generally `advancedDetailsXXX` template), these should be auxiliary functions that could be called from other templates, like `advancedDetailsHazards` or the more general `scenarioFormAdvancedInfo`. By doing this we could use this functions in other templates.

**Comment by diego2013** [comment]
Seems to be where it's siblings are. ScenarioCompleteForm uses `{{> advancedDetailsHazards}}` instead of using it's own "duplicated" template.

**Comment by diego2013** [comment]
The code was duplicated because was being used in the ScenarioComplete template but outside the individual templates for the "advanced info" panels. The information being displayed (count for entries) was duplicated, though.  

So I left the counters inside each template and for the parent template I removed these numbers, since the information was redundant anyway. I added a better style, to remark this as a header section and not just a label.
